docs/section22/Rule22-10.md

```diff
@@ -0,0 +1,67 @@
+
+# CHW&CW - Rule 22-10  
+
+**Schema Version:** 0.0.10  
+**Mandatory Rule:** True  
+**Rule ID:** 22-10  
+**Rule Description:** For Baseline chilled water system with cooling capacity less than 300ton, the secondary pump shall be modeled as riding the pump curve. For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary pump shall be modeled with variable-speed drives.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.10 Chilled-water pumps (System 7, 8, 11, 12 and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, 13, 7b, 8b, 11b, 12b, 13b.
+2. B-RMR is modeled with primary-secondary configuration.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. get_primary_secondary_loops()
+
+**Applicability Checks:**  
+
+- **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): NEXT_APPLICABILITY_CHECK`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+- **Check 2:** Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - Check if B-RMR is modeled with primary-secondary configuration, continue to rule logic: `if LEN(primary_secondary_loop_dictionary) != 0: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each pump in B_RMR: `for pump_b in B_RMR.ASHRAE229.pumps:`
+
+  - Save loop served by pump and pump to a dictionary: `loop_pump_dictionary[pump_b.loop_or_piping].append(pump)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - Add chiller rated capacity to CHW loop total capacity: `chw_loop_capacity_dict[chw_loop_b] += chiller_b.rated_capacity`
+
+- For each primary CHW loop: `for loop_b_id in primary_secondary_loop_dictionary["PRIMARY"]:`
+
+  - Check if CHW loop total cooling capacity is less than 300 tons, set target secondary pump type to riding the pump curve: `if chw_loop_capacity_dict[loop_b_id] < 300 * CONVERSION(TON_TO_BTUH): target_secondary_pump_type == "FIXED_SPEED"`
+
```

Comment:
SAME CCOMEMNT AS #301 THE CHILLER CAPACITY will be in W in the RMR. Thee conversion should be W to Tons and the check should be for capacity >300 Tons. Correct?

Reply:
The 300ton is from 90.1 table, and it is converted to Btuh as we assumes the chiller rated capacity in IP unit is Btuh. @weilixu please confirm if this will work.

Reply:
I don't think we should assume the IP units for RDS. The RDS is reading the RMR which is in SI units. The RDS should identify the required conversion from the units listed in the schema to the units required for the RDS verification- i.e. the units in Standard 90.1